src/module/item/application/service/item.service.ts

```diff
@@ -1,12 +1,25 @@
 import { Injectable, Inject } from '@nestjs/common';
-import {
-  ITEM_REPOSITORY,
-  IItemRepository,
-} from '../repository/item.repository.interface';
+import { ITEM_REPOSITORY } from '../repository/item.repository.interface';
+import { Item } from '../../domain/item.entity';
+import { ItemRepository } from '../../infrastructure/persistence/item.repository';
+import { Order } from '../../../../../src/module/order/domain/order.entity';
 
 @Injectable()
 export class ItemService {
   constructor(
-    @Inject(ITEM_REPOSITORY) private itemRepository: IItemRepository,
+    @Inject(ITEM_REPOSITORY) private itemRepository: ItemRepository,
```

Comment:
the interface IItemRepository should be the type

Reply:
Done